examples/source-rouge.adoc

```diff
@@ -39,3 +40,20 @@ fn main() {
     println!("Hello stretched World!");
 }
 ----
+
+== Rouge Reatures
+
+// TODO docinfo not loaded
+// TODO linenums feature affects listing size
+
+This is broken
```

Comment:
Actually, it's a limitation of Rouge: https://github.com/rouge-ruby/rouge/issues/264
I think we shouldn't bother too much.

Reply:
I'm not going to do it.